addon/models/resource.js

```diff
@@ -415,8 +418,14 @@ Resource.reopenClass({
     for (let i = 0; i < attrs.length; i++) {
       prototype[attrs[i]] = {};
     }
+
     const instance = this._super(prototype);
     if (properties) {
+      if (properties.hasOwnProperty('id')) {
+        // JSONAPI ids MUST be strings.
+        properties.id = properties.id.toString();
```

Comment:
Mmmm... meh... It _was_ kinda fitting when we were thinking about adding code to the serializer for casting to string.. but since we're not doing that (it's handled by `Resource.create`) I'm against adding code in the serializer that does nothing else but warn the user. It's not really our problem that the server is broken, and as far as we can cope with the payload, fine. But we're not "your servers mommy" 😃